Log/v1.5_Changes.md

```diff
@@ -0,0 +1,50 @@
+# Version 1.5 Changes
+
+For many many reasons, we're making some significant changes to the program that will go into effect starting June 19, 2017.
+
+This document outlines the planned changes from a high level. More details to follow.
+
+## What's going away
+
+- Stats (i.e. most of what the "game" formerly was)
+- Levels
+- Wandering in the wilderness (lack of learning tracks)
```

Comment:
I would just call this "lack of sequencing". Another way some people will read this is "self-directed learning". It's a matter of how we want to frame it.